README.md

```diff
@@ -70,6 +70,17 @@ Once started and if you create a Zabbix machine run:
 ### Bulk user creation
 Run:  `BulkUsersCreation.sh` 
 
+
+### Wanna test autoscaling?
+
+Enjoy your nodes automagically enrolled in Zabbix.
+
+    juju ssh clearwater-bono/0
+    sudo apt-get install -y -qq stress
+    stress --cpu 1 --io 2 --vm 2 --vm-bytes 512M --timeout 600
```

Comment:
I would prefer metaswitch to document the clearwater sipp usage to stress using a real workload engine. Then CABS